semgrep/skip_list.txt

```diff
@@ -1,7 +0,0 @@
-# This configuration file is used by semgrep-core -pycheck (as well
```

Comment:
did we not need this anymore?

Reply:
I think this was for the `foo_.pyi` interface files which we removed a while ago.